wordpress-mobile/WordPress-iOS
# New Posts / Pages status and author filter bar

Fixes #8823. This PR replaces the post status dropdown filter that was in the navigation bar of the Posts and Pages screens, as well as the author segmented control that was in the Posts screen. 

The most recent reference designs can be found here: https://github.com/wordpress-mobile/WordPress-iOS/issues/8823#issuecomment-372324567. I'm currently referencing a Gridicons branch for the new multiple users icon, but I'll update this PR once that's merged.

![filter-tabs](https://user-images.githubusercontent.com/4780/37338298-04a610c6-26af-11e8-889f-70b767fa61b4.gif)

Note that self-hosted (non-Jetpack) sites will currently show a placeholder Gravatar.

Apologies for the large PR!

**To test:**

* For both Posts and Pages, ensure that the filter bar displays correctly.
* Ensure that you can switch between the tabs, and that the content is updated as expected.
* Check that if you leave the posts / pages screen and then return, your selection is restored.
* Check that the navigation bar dropdown is no longer presenter.
* Check that the search bar still works as expected.
